features/project/project_controller.rb

```diff
@@ -18,6 +18,13 @@ class ProjectController < AuthenticatedControllerBase
 
       repo = FastlaneCI::GitRepo.new(git_config: project.repo_config, provider_credential: current_github_provider_credential)
       current_sha = repo.most_recent_commit.sha
+      manual_triggers_allowed = project.job_triggers.any? { |trigger| trigger.type == FastlaneCI::JobTrigger::TRIGGER_TYPE[:manual] }
```

Comment:
Might be nice to (dynamically?) add some query methods to `JobTrigger` such as `job_trigger.manual?` so that this can be re-written as `project.job_triggers.any?(&:manual?)`

Reply:
That would be cool! Mind if i tackle this in a separate PR?

Reply:
👍 sounds good!